pkg/domain/infra/abi/images.go

```diff
@@ -334,46 +334,124 @@ func (ir *ImageEngine) Push(ctx context.Context, source string, destination stri
 	return pushError
 }
 
-// Transfer moves images from root to rootless storage so the user specified in the scp call can access and use the image modified by root
-func (ir *ImageEngine) Transfer(ctx context.Context, scpOpts entities.ImageScpOptions) error {
-	if scpOpts.User == "" {
+// Transfer moves images between root and rootless storage so the user specified in the scp call can access and use the image modified by root
+func (ir *ImageEngine) Transfer(ctx context.Context, source entities.ImageScpOptions, dest entities.ImageScpOptions, parentFlags []string) error {
+	if source.User == "" {
```

Comment:
What happens if dest.User="USER" and source.User="", would that mean root or current user to USER?

Reply:
I can make this more clear, I think only source gets a user assigned to it and the destination user is the "opposite" so if source.User = charliedoern then dest.User = root and vice versa

Reply:
What if I do

podman image scp challierdoern@localhost::alpine dwalsh@loalhost::

Reply:
I did not think of that, I can add user to user options.